pr_work_done.rb

```diff
@@ -58,79 +60,71 @@ def date_of_next(day)
   date + delta
 end
 
-#find next wedenesday, set boundries
-right_bound = date_of_next "Wednesday"
+# find next wedenesday, set boundries
+right_bound = date_of_next 'Wednesday'
 left_bound = right_bound - 7
-since = (right_bound - (number_of_weeks_to_show * 7))
+# since = (right_bound - (number_of_weeks_to_show * 7))
 
 all_merged_pulls = []
 all_closed_pulls = []
 comments = []
 members_of_organisation = {}
 
-#gather all commments / merges / closed in our time range (since)
+# gather all commments / merges / closed in our time range (since)
 repos.each do |repo|
-  closed_pr_information_cache = util.fetch_async("#{options[:namespace]}/#{repo}", git_options={:state=>'closed'}, filter=[:issue_comments], limit={:attribute=>'closed_at', :date=>since})
-  #closed prs
+  closed_pr_information_cache = util.fetch_async("#{options[:namespace]}/#{repo}} ")
```

Comment:
namespace and repository information should be obtained from the modules.json entities